client/css/style.css

```diff
@@ -762,15 +762,15 @@ button {
 }
 
 #chat .messages {
-	display: table;
-	table-layout: fixed;
-	width: 100%;
 	padding: 10px 0;
 }
 
 #chat .msg {
 	word-wrap: break-word;
 	word-break: break-word; /* Webkit-specific */
+	display: flex;
+	overflow: hidden;
+	position: relative;
```

Comment:
I could not figure out why these last 2 lines (`overflow: hidden;` and `position: relative;`) are necessary. It seems like disabling them doesn't do anything for me.
Do you remember, by any chance?